src/libs/actions/Task.js

```diff
@@ -88,6 +88,9 @@ function createTaskAndNavigate(parentReportID, title, description, assignee, ass
                 ...optimisticTaskReport,
                 pendingFields: {
                     createChat: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
+                    ...(title && {reportName: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD}),
+                    ...(description && {description: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD}),
+                    ...(assigneeAccountID && {managerID: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD}),
```

Comment:
I think we should set these all pending on creation. For example in the test videos the assignee field isn't grayed out because it was unset, but I think the whole task should show as grayed out because the entire thing is pending creation.